scripting/pugsetup.sp

```diff
@@ -2237,3 +2244,30 @@ stock bool PermissionFromString(const char[] permissionString, Permission& p,
 
   return true;
 }
+
+public int GetTotalPlayers()
+{
+    int count;
+    LoopClients(client)
+    {
+        count++;
+    }
+    
+    return count;
+}
+
+public void OnClientDisconnect(int client)
+{
+    if (g_GameState == GameState_Live)
+    {    
+        if(GetTotalPlayers() <= 2)
+        {
+            GetConVarBool(g_EndGameOnLastPlayerInServerCvar);
+            {
+                ServerCommand("sm_forceend");
+                ReadSetupOptions();
+                SetupFinished();
+            }
```

Comment:
this isn't doing what you think.

consider:
```
if (g_EndGameOnLastPlayerInServerCvar.BoolValue)
{
    ServerCommand("sm_forceend");
    ReadSetupOptions();
    SetupFinished();
}
```

Reply:
thx, ive updated